src/components/ArticleListCard.jsx

```diff
@@ -9,14 +13,13 @@ function ArticleListCard({ article }) {
     article_img_url,
     comment_count,
   } = article;
+  const handleClick = () => {
+    navigate(`/article/${article_id}`);
+  };
   return (
     <div>
-      <div className="card-container">
-        <div>
-          <div>⬆️</div>
-          <div>{votes}</div>
-          <div>⬇️</div>
-        </div>
+      <div onClick={handleClick} className="card-container">
```

Comment:
This has implications for accessibility as DIVs are not clickable by default. If you wanted to do this, you would need to add extra info like a role or data attribute. Take a look here if you wish to keep using this approach: https://eng.wealthfront.com/2020/10/01/building-a-truly-accessible-clickable-div/